src/many-modules/src/_4_events.rs

```diff
@@ -637,6 +645,20 @@ mod test {
         assert_eq!(Into::<Vec<u8>>::into(t2), (v - 1).to_be_bytes());
     }
 
+    #[test]
+    fn event_info_addresses() {
```

Comment:
Test inner resolution with a multisig submit tx.

Reply:
We don't have resolution for `inner` yet. It's... very hard.